src/elm/Page/Community/New.elm

```diff
@@ -502,13 +512,14 @@ validateModel accountName model =
                         , logoUrl = logoUrl
                         , name = name
                         , description = model.description
-                        , subdomain = Just model.subdomain
+                        , subdomain = model.subdomain ++ ".cambiatus.io"
```

Comment:
one day we will allow communities to go full whitelabel and use their own domains!

Reply:
Should we account for that right now though?

Reply:
no, not at all! just a comment (: